cmake/modules/FindMPI.cmake

```diff
@@ -136,7 +136,9 @@ CHECK_COMPILER_SETUP(
   ${MPI_CXX_LIBRARIES} ${MPI_Fortran_LIBRARIES} ${MPI_C_LIBRARIES}
   )
 
-IF(NOT MPI_UNDERLINKAGE_OK)
+IF(NOT MPI_UNDERLINKAGE_OK AND MPI_CXX_LIBRARIES)
```

Comment:
Would you mind changing this to `AND NOT "${MPI_CXX_LIBRARIES}" STREQUAL ""`?

Reply:
updated.